src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -120,62 +157,428 @@ def _load_iges(self):
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   1
+
+        Or using the entity name:
+
+        >>> line1 = mapdl.geometry.lines["line 1"]
+        >>> line1
+        PolyData (0x147d5b220)
+          N Cells:    1
+          N Points:   100
+          N Strips:   0
+          X Bounds:   0.000e+00, 0.000e+00
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   1
+
+        You can use this to iterate over the different elements:
+
+        >>> points = mapdl.geometry.lines
+        >>> for each_line in points:
+                print(each_line)
+        PolyData (0x147d5b220)
+          N Cells:    1
+          N Points:   100
+          N Strips:   0
+          X Bounds:   0.000e+00, 0.000e+00
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, -3.180e-03
+          N Arrays:   1
+        ...
+
+        """
+        mb = Multiblock(self._lines)
+        # Setting names
+        for ind, each_block in enumerate(mb):
+            mapdl_index = int(each_block["entity_num"][0])
+            mb.set_block_name(index=ind, name=f"line {mapdl_index}")
+        return mb
+
+    def get_lines(
+        self, return_as_list: bool = False
+    ) -> Union[pv.PolyData, List[pv.PolyData]]:
+        """Obtain line geometry
+
+        Obtain the active lines as a :class:`pyvista.PolyData` object or
+        a list of :class:`pyvista.PolyData`
+
+        Parameters
+        ----------
+        return_as_list : bool, optional
+            Return the lines in a list, by default False
+
+        Returns
+        -------
+        Union[pv.PolyData, List[pv.PolyData]]
+
+        Examples
+        --------
+
+        Return a single merged mesh.
+
+        >>> line_mesh = mapdl.geometry.get_lines()
+        >>> line_mesh
+        PolyData (0x14917e740)
+          N Cells:    45
+          N Points:   4500
+          N Strips:   0
+          X Bounds:   -1.588e-02, 1.588e-02
+          Y Bounds:   -7.620e-03, 1.778e-02
+          Z Bounds:   -3.180e-03, 1.524e-02
+          N Arrays:   1
+
+        Return a list of lines as indiviudal grids
+
+        >>> lines = mapdl.geometry.get_lines(return_as_list=True)
+        >>> lines
+        [PolyData (0x1492ee7a...rrays:   1, PolyData (0x1491a404...rrays:   1, PolyData (0x1491068c...rrays:   1, PolyData (0x14910662...rrays:   1, PolyData (0x14910632...rrays:   1, PolyData (0x1492eeb0...rrays:   1, PolyData (0x1492eec8...rrays:   1, PolyData (0x1492eee0...rrays:   1, PolyData (0x1492eef8...rrays:   1, PolyData (0x1492ef10...rrays:   1, PolyData (0x1492ef28...rrays:   1, PolyData (0x1492ef40...rrays:   1, PolyData (0x1492ef58...rrays:   1, PolyData (0x1492ef70...rrays:   1, ...]
+
+        """
+        if return_as_list:
+            return self._lines
+        else:
+            return merge_polydata(self._lines)
+
+    @property
+    def areas(self) -> pv.MultiBlock:
+        """Obtain the areas geometry
```

Comment:
```suggestion
        """Geometry of the areas as a :class:`pyvista.MultiBlock` object.
```
Is there only one geometry of the area as written?  You can delete line 478 and put all the needed information in one description line.

Reply:
In theory first line is a short description and the following text block more detailed. But I did not have much to write there.

I will keep the return object descriptor out of the short summary. It seems out of place otherwise.